src/languages/lambda/main.js

```diff
@@ -0,0 +1,6 @@
+import Parser from "./code";
+
+export default function main(code) {
+  var ast = Parser.parse(Parser.TokenStream(Parser.InputStream(code)));
+  return ast;
+}
```

Comment:
I would ditch this file and move this code into `code.js`. You can name the function `parseString(code)` or something like that.